config/routes.rb

```diff
@@ -1,3 +1,16 @@
 Rails.application.routes.draw do
+  get 'users/new'
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "static_pages#home"
+
+  get '/help' => 'static_pages#help'
+
+  get '/home' => 'static_pages#home'
+
+  get '/about' => 'static_pages#about'
+
+  get '/contact' => 'static_pages#contact'
+
+  get '/signup' => 'users#new'
```

Comment:
- using double quote in this file
- Change all: => `get "/help": "static_pages#help"`